extensions/amp-experiment/1.0/test/test-amp-experiment.js

```diff
@@ -86,6 +86,17 @@ describes.realWin('amp-experiment', {
     experiment.element.appendChild(child);
   }
 
+  function stubAllocateVariant(sandbox, config) {
```

Comment:
I'm curious if we can remove the stub by setting the test window url to enforce the experiment value?

Reply:
Gonna keep the stub, so we can isolate the code to just running `amp-experiment.js` 😄